src/test/java/com/fittracker/fittracker/dataprovider/Error.java

```diff
@@ -0,0 +1,10 @@
+package com.fittracker.fittracker.dataprovider;
+
+import static java.lang.String.format;
+
+public class Error {
+
+    public static String errorResponseString(String field, String message) {
```

Comment:
let's just add the method to `ProductControllerValidationTest`, it's not used anywhere else